src/server/lib/audit-report.js

```diff
@@ -1,876 +1,118 @@
-/* eslint camelcase: 0 */
-
-/*
---------------------------------------------------------------------------------
--                           lib/audit-report.js
---------------------------------------------------------------------------------
-
-*/
-const _ = require('lodash')
+const { format } = require('date-fns')
```

Comment:
whoa has `date-fns` been in our bundle this whole time? maybe we should just stick to moment, it's used in many more places

Reply:
You're definitely right.  I just cloned this from the previous version of the code.  I'll split off a follow-up!

Reply:
I had to rebase, so I snuck this in after all.  👍